.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,25 @@
+name: 🐛 Bug
+description: Report an issue to help improve the project.
+labels: ['🛠 goal: fix']
```

Let's use basic tags

```suggestion
labels: [bug, triage]
```